sockeye/utils.py

```diff
@@ -167,7 +167,16 @@ def load_params(fname: str) -> Tuple[Dict[str, mx.nd.NDArray], Dict[str, mx.nd.N
     for k, v in save_dict.items():
         tp, name = k.split(':', 1)
         if tp == 'arg':
-            arg_params[name] = v
+            """TODO(fhieber):
```

Comment:
how about creating a 1.19 branch with this splitting removed, that we can use for accumulating other changes and also for not forgetting do remove this.

Reply:
given the benefits of the change lets get it in right now. I will keep it on my list to remove the logic/this TODO once we bump to 1.19.